src/ActionHandler/FindRepositoryThroughOrganizationHandler.php

```diff
@@ -3,25 +3,25 @@
 namespace OpenCatalogi\OpenCatalogiBundle\ActionHandler;
 
 use  CommonGateway\CoreBundle\ActionHandler\ActionHandlerInterface;
-use OpenCatalogi\OpenCatalogiBundle\Service\FindGithubRepositoryThroughOrganizationService;
+use OpenCatalogi\OpenCatalogiBundle\Service\FindRepositoryThroughOrganizationService;
 
 /**
- * Executes a the FindGithubRepositoryThroughOrganizationService that loops through organizations (https://opencatalogi.nl/oc.organisation.schema.json)
+ * Executes a the FindRepositoryThroughOrganizationService that loops through organizations (https://opencatalogi.nl/oc.organisation.schema.json)
  * and tries to find a opencatalogi.yaml on github with its organization name to update the organization object with that fetched opencatalogi.yaml data.
  */
-class FindGithubRepositoryThroughOrganizationHandler implements ActionHandlerInterface
+class FindRepositoryThroughOrganizationHandler implements ActionHandlerInterface
```

Comment:
Keep in mind that you risk that existing actions that are not patched properly will trigger 500 errors on this change. Please update online environments very carefully.